atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/ThreeNodeCassandraClusterOperations.java

```diff
@@ -51,6 +55,9 @@ public void replicateSystemAuthenticationDataOnAllNodes()
         if (!systemAuthenticationKeyspaceHasReplicationFactorThree()) {
             setReplicationFactorOfSystemAuthenticationKeyspaceToThree();
             runNodetoolRepair();
+            if (!systemAuthenticationKeyspaceHasReplicationFactorThree()) {
+                throw new IllegalStateException("Replication factor is still not 3!");
+            }
```

Comment:
Not sure we want to add this. It adds quite a bit of extra overhead since we need to get docker to execute a network command. The overhead is already quite big.